packages/server/assets/workspacesCore/typedefs/workspaces.graphql

```diff
@@ -276,7 +276,7 @@ type Workspace {
   """
   Billing data for Workspaces beta
   """
-  billing: WorkspaceBilling! @hasWorkspaceRole(role: MEMBER)
+  billing: WorkspaceBilling
```

Comment:
y

Reply:
I could set it as requiring workspace guest, but the important bits are:

- Opting not to change frontend query based on role
- Returning `null` for billing if user is guest

We could also just return the billing information anyway for guests and continue not to show it on the frontend, but this seemed intentional to not expose that information to guests.

Reply:
Ok, that's fine.

As an FYI - you could've also set `errorPolicy: 'all'` in the FE to prevent the query from not loading all of the other fields because of the auth issue in regards to this specific field, this is also what we do for `invitedTeam`, another admin only field. The benefit there would be that the API would be clearer - instead of just returning null, it'd actually throw saying you don't have access, plus there's the directive in the typedef.

Reply:
If there is an easy schemantically correct frontend fix for this I'd like to do that. @fabis94 @cdriesler pls coordinate on this

Reply:
@gjedlicska opting to do the `errorPolicy` thing